CHANGELOG.md

```diff
@@ -1,3 +1,269 @@
+* [#165:](https://github.com/pysal/spopt/pull/165) remove draft PDF action 
+* [#164:](https://github.com/pysal/spopt/issues/164) remove draft_pdf action 
+* [#163:](https://github.com/pysal/spopt/pull/163) Fixes documentation 
+* [#162:](https://github.com/pysal/spopt/issues/162) Import clauses in documentation are wrong throughout the examples 
+* [#161:](https://github.com/pysal/spopt/pull/161) correct paper indent & add JOSS badge 
+* [#160:](https://github.com/pysal/spopt/pull/160) Bump actions/upload-artifact from 1 to 2.2.3 
+* [#159:](https://github.com/pysal/spopt/pull/159) Add paper 
+* [#158:](https://github.com/pysal/spopt/pull/158) include the paper in the git-based repository together with the software 
+* [#153:](https://github.com/pysal/spopt/pull/153) Bump actions/create-release from 1 to 1.1.4 
+* [#156:](https://github.com/pysal/spopt/pull/156) Bump actions/upload-release-asset from 1 to 1.0.2 
+* [#157:](https://github.com/pysal/spopt/pull/157) Bump conda-incubator/setup-miniconda from 2 to 2.1.1 
+* [#155:](https://github.com/pysal/spopt/pull/155) Bump codecov/codecov-action from 1 to 1.5.0 
+* [#154:](https://github.com/pysal/spopt/pull/154) Bump actions/cache from 2 to 2.1.5 
+* [#152:](https://github.com/pysal/spopt/pull/152) Update docs for random region 
+* [#147:](https://github.com/pysal/spopt/issues/147) Support for RandomRegions? 
+* [#150:](https://github.com/pysal/spopt/issues/150) Add support for randomregions 
+* [#151:](https://github.com/pysal/spopt/pull/151) Adding random regions 
+* [#149:](https://github.com/pysal/spopt/pull/149) test 
+* [#148:](https://github.com/pysal/spopt/pull/148) update region-k-means notebook 
+* [#146:](https://github.com/pysal/spopt/pull/146) revise ward in notebooks 
+* [#145:](https://github.com/pysal/spopt/pull/145) revised notebook for azp 
+* [#144:](https://github.com/pysal/spopt/pull/144) Bump actions/cache from v2 to v2.1.4 
+* [#143:](https://github.com/pysal/spopt/pull/143) update dev docs link 
+* [#142:](https://github.com/pysal/spopt/pull/142) revise skater in notebooks 
+* [#140:](https://github.com/pysal/spopt/issues/140) Tutorials link in readme leads to a 404 error 
+* [#141:](https://github.com/pysal/spopt/pull/141) update tutorials link in README 
+* [#138:](https://github.com/pysal/spopt/issues/138) fresh cut release? 
+* [#139:](https://github.com/pysal/spopt/pull/139) bump version 0.1.0 -> 0.1.1 
+
+
+<a name="pysal"></a>
+### pysal
+* [#1223:](https://github.com/pysal/pysal/issues/1223) Spatial Weights issue, Points geometry not allowed 
+* [#1220:](https://github.com/pysal/pysal/issues/1220) Could not load shared object file: llvmlite.dll 
+* [#1218:](https://github.com/pysal/pysal/pull/1218) 2.4.0 
+
+<a name="contributors"></a>
+## Contributors
+
+Many thanks to all of the following individuals who contributed to this release:
+
+
+ - Arfon Smith
+ - Dani Arribas-Bel
+ - Eli Knaap
+ - Elliott Sales De Andrade
+ - James Gaboardi
+ - Jkoschinsky
```

Comment:
Should we have Julia's GitHub handle or her name? @Jkoschinsky
* same for:
  * Mragank @Mgeeeek
  * Xin (Selena) vs. Xin @xf37

And we should add @ljwolf.

Reply:
I'll prolly change my weird username, sorry for the trouble :'(

Reply:
> Should we have Julia's GitHub handle or her name? @jkoschinsky
> 
>     * same for:
>       
>       * Mragank @MgeeeeK
>       * Xin (Selena) vs. Xin @xf37
> 
> 
> And we should add @ljwolf.

The contributor list is autogenerated in https://github.com/pysal/pysal/blob/master/tools/110-gitcount-tables.ipynb

There is a mapping to link an id to a preferred name. PRs to adjust would be welcomed.